### ISSUE TITLE
GPS stopped working on 7a after update to 2024042100

### ISSUE THREAD

**Issue by cascading-jox** [GrapheneOS/os-issue-tracker#3469]
I just updated to [2024042100](https://grapheneos.org/releases#2024042100) and the GPS stopped working. Before I was not using the reroute to OS option but now neither of the GPS routing options have any effect. On google maps, no indicator for my position appears. Is this a known issue?

**Comment by Lppsoeht** [comment]
The same happened to me a few times after updates on p6 pro, sometimes it works again, no apparent reason.

**Comment by haary** [comment]
I experience the same issue on the 7a  after the latest update. I haven't changed any setting , before Google Maps GPS was working, after the update it's not.

**Comment by Lppsoeht** [comment]
> @Lppsoeht Your device is configured incorrectly. It's not an OS issue.

I don't use sandboxed google services. I have everything else location-related enabled except for WiFi and Bluetooth scanning.
So it doesn't work indoors? Next time I'll be out of the shelter I'll update u guys

**Comment by cascading-jox** [comment]
@thestinger 

> Unclear what you mean by either of the GPS routing issues.

I remember when I first flashed the OS I still got some kind of GPS signal with OS rerouting enabled even though it was slow and not accurate. Now when trying either configuration I get no signal. 

> If you've disabled PSDS or SUPL, that will severely hinder GNSS-based location. You cannot expect GNSS-based location to work indoors since it requires a satellite signal and would need to configure network-based location for that, which is not enabled by default but can be used.

I haven't changed any settings and I don't expect it to work indoors.

The thing is that the phone does not find any signal at all. Usually it first finds some satellites after a second or two, then you get an inaccurate position and after a little longer you get a GPS lock. In my case it is stuck on the first step. 

> GrapheneOS has the same GNSS-based location approach as the stock OS. It's not a worse implementation. The difference is that they get you to enable network-based location by default and Play services has the required permissions by default. This is available on GrapheneOS if you want it.

Ok, I wouldn't assume anything less. It is a fantastic OS :)

Basically what happened is that I had disabled OS rerouting and GPS was working perfectly fine and then I updated to `2024042100` (from `2024040900` or `2024042000` I think) and then after the update, GPS was not working anymore as described. 

If there is no bug but a configuration issue (however that happened), I assume a factory reset would solve the issue?

**Comment by Lppsoeht** [comment]
Can confirm that outdoors works, it takes 10 seconds or so

**Comment by cascading-jox** [comment]
An update: I haven't manually changed any configuration or updated anything but yesterday it suddenly started working again, not sure what changed.
